src/Network/Receive.pm

```diff
@@ -2406,4 +2406,48 @@ sub hat_effect {
 	}
 }
 
+##
+# 096D <size>.W { <index>.W }*
+# @author [Cydh]
+##
+sub merge_item_open {
+	my ($self, $args) = @_;
+	my $num = ($args->{length} - 4) / 2;
```

Comment:
Maybe just use `$num = scalar @mergeItemList` here, displays intent better, and `length` would not be needed at all anywhere.